packages/terra-core-docs/src/terra-dev-site/doc/tag/TagList.2.doc.mdx

```diff
@@ -0,0 +1,49 @@
+import { Badge } from 'terra-tag/package.json?dev-site-package';
+
+import TagListCollapsed from './example/TagListCollapsed?dev-site-example';
+
+import TagListSmall from './example/TagListSmall?dev-site-example';
+
+import TagPropsTable from 'terra-tag/lib/TagList?dev-site-props-table';
+
+<Badge />
+
+# Terra Tag List
+
+The Tag List is a subcomponent of main component `tag` which is used for grouping tabs together in a list to enable keyboard navigation using arrow keys. Navigation from one list to another can be done using `tab` key and within the list, navigation can be done using arrow keys, `right key` to move to next tag and `left key` to move to previous tag.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-tag-list`
+
+<!-- AUTO-GENERATED-CONTENT:START Peer Dependencies -->
+## Peer Dependencies
+
+This component requires the following peer dependencies be installed in your app for the component to properly function.
+
+| Peer Dependency | Version |
+|-|-|
+| react | ^16.8.5 |
+| react-dom | ^16.8.5 |
```

Comment:
Can we add React-Intl as a peer dependency in both Tag About page and here as well

Reply:
Updated :) https://github.com/cerner/terra-core/pull/3854/commits/8ded50f47258067e34fd8a01eed5eb3b47b92fd7#diff-43d71167fbce146e41d65538d63d237642369f5d98c53c55d8294d764b82ddfb